Dockerfile

```diff
@@ -13,4 +13,7 @@ RUN echo "/home/octave/bioformats_package.jar" >> /home/octave/javaclasspath.txt
 
 
 RUN echo "pkg install bioformats-octave-$VERSION.tar.gz" | octave
+
+RUN wget https://downloads.openmicroscopy.org/images/OME-TIFF/2016-06/tubhiswt-3D.zip
```

Comment:
We don't have anything in place for utm_source. I had forgotten the specifics of --user-agent so let's stick with that.